fix hardcoded path to entertain-srv binary

vKnmnn opened the pull request:
As part of getting my split-brain hue setup to work, i need the paths to the binary to not be hardcoded.

This shouldn’t make any difference for non-split setups.

mariusmotea commented:
Looks good for me. Just for my knowledge, why you need this change?

juanesf commented:
pwd works for me in openwrt, can it be pwd instead of cwd?

mariusmotea commented:
@juanesf cwd looks to be a bash command like pwd, but is not. It is a variable declared in python script header. To test it start python interpreter, import sys module and run the following command:

https://github.com/diyhue/diyHue/blob/ed1d962731eeb78bd50c649a3c57eae6a3bc1be9/BridgeEmulator/HueEmulator3.py#L60

vKnmnn commented:
@mariusmotea 

I have two instances of diyHue running. They run on different interfaces with different MAC addresses, so they need their own certificate. 

So i copied /opt/hue-emulator to /opt/hue-emulator-main, changed paths in the service file.
Then i relaunched the install script and ended up with a second instance, which i moved to /opt/hue-emulator/secondary. Upon starting the services i noted some errors regarding entertainment.py, to which the location was hardcoded.

mariusmotea commented:
Please move this to dev branch. From dev i will move to master. Dev branch must always be ahead of master.
